website/blog/2021-02-08-64updates.md

```diff
@@ -0,0 +1,170 @@
+
+![rn-gallery-gif](assets/gallery_demo_1.gif)
+
+Much like the [XAML Controls Gallery](https://www.microsoft.com/p/xaml-controls-gallery/9msvh128x2zt) for WinUI, we intend to have the React Native Gallery be available not only as a proving ground for our own work, but also a place where native and JavaScript components alike can be seen in action in the Windows ecosystem. This includes providing lightweight code snippets on each page for developers to copy out and use in their own scenarios.
+
+## 🧪 Try it out now!
+To get your hands on a copy of the app while we wait for final Store approval, head on over to the repo and check out the [certified install on our 0.64 branch](https://github.com/microsoft/react-native-gallery/tree/v0.64.0-preview.5/windows/rngallery_1.0.2.0_Win32_Test).
+
+Alternately, you can get the latest state of the app and the source to test things out directly on our [main repo branch](https://github.com/microsoft/react-native-gallery).
+
+# ✨ Developer Goodness
+As with many of our releases, there's always a number of developer and "under the covers" upgrades we make to the framework as a whole. These usually includes things like performance improvements, better API parity and of course general bug fixes, but let's take a look at some of the notable items available this release.
+
+## 🎉 API parity highlights
+
+More component properties supported by the other platforms now work on Windows! Check them out.
+
+### ⚙ `Platform.version`
+The core API `Platform` has now been extended to include Windows. This means you can improve your styles and refine your Windows app experience.
+
+``` js
+import { Platform, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,    
+    ...Platform.select({
+      Windows: {
+        backgroundColor: 'blue'
+      },
+      default: {
+        // fallback for other/unspecified platforms
+        backgroundColor: 'grey'
+      }
+    })
+  }
+});
+```
+
+### ⚙ More `Text` properties
+The `Text` component properties `backgroundColor`, `border`, and `textTransform` are now available on Windows and apply to our native text controls as you'd expect.
+
+- **backgroundColor** will set the backfill/plate color behind your text.
+- **border** encompasses all the border functionalities, like borderWidth, borderColor, etc.
+- **textTransform** can force all strings lowercase, uppercase, or capitalization depending.
+
+```js
+import React from "react";
+import { Text, StyleSheet } from "react-native";
+
+const TextTest = () => {
+  return (
+    <Text style={styles.newTextProperties}>
+      some text goes here
+    </Text>
+  );
+};
+
+const styles = StyleSheet.create({
+  newTextProperties: {
+    color: "white",
+    backgroundColor: "blue",
+    borderColor: "black",
+    borderWidth: 1,
+    textTransform: "capitalize"
+  }
+});
+
+export default TextTest;
+```
+
+### ⚙ `AccessibilityInfo`

```

Comment:
@acoates-ms can you double-check that I'm encompassing the changes to this API this release accurately? (or if you think there should be more detail here) please let me know!

Reply:
Looks good.